docs/reference/field-types/relationship.md

```diff
@@ -118,3 +118,15 @@ _players: {
   withRelationships: [ '_specialties' ]
 }
 ```
+
+In case of double nesting e.g. _specialties piece has a `_photo` relationship field with another piece `@apostrophecms/image`, then `_photo` can be accessed as `_specialties._photo`
```

Comment:
Please modify this sentence slightly -'In the case of double nesting e.g., ...` and add a period at the end.

Reply:
Updated, is that what you wanted ? or do you want me to change wording ?

Reply:
Add "the" in front of "case" at the beginning of the sentence, and eliminate the parentheses, please. Thanks!

Reply:
Updated👍